contrib/gh-runner/gh-runner.sh

```diff
@@ -26,6 +26,7 @@ curl -o actions-runner-linux-x64-2.296.2.tar.gz -L https://github.com/actions/ru
 echo "34a8f34956cdacd2156d4c658cce8dd54c5aef316a16bbbc95eb3ca4fd76429a  actions-runner-linux-x64-2.296.2.tar.gz" | shasum -a 256 -c
 tar xzf ./actions-runner-linux-x64-2.296.2.tar.gz
 ./config.sh --url https://github.com/dgraph-io/dgraph --token $TOKEN
+sudo chown -R $USER:$USER /home/ubuntu/actions-runner/_work # fixes the test cruft cleanup issue
```

Comment:
@joshua-goldstein @dshekhar95 this is the correct way to fix it at the infrastructure layer.